official/resnet/resnet_run_loop.py

```diff
@@ -381,11 +381,9 @@ def resnet_main(flags, model_function, input_function, shape=None):
           'dtype': flags.dtype
       })
 
-  if flags.benchmark_log_dir is not None:
-    benchmark_logger = logger.BenchmarkLogger(flags.benchmark_log_dir)
-    benchmark_logger.log_run_info('resnet')
-  else:
-    benchmark_logger = None
+  logger.config_benchmark_logger(flags.benchmark_log_dir)
+  benchmark_logger = logger.get_benchmark_logger()
```

Comment:
Why not have this rolled into one line? Config and return the logger in one call, with separate interfaces as necessary.

Reply:
Done.